class.jetpack-debugger.php

```diff
@@ -18,6 +18,12 @@ private static function is_jetpack_support_open() {
 		}
 	}
 
+	private static function what_jetpack_plan() {
+		$plan = Jetpack::get_active_plan();
+		$plan = $plan['class'];
```

Comment:
I agree.  The way `get_active_plan()` is written it makes it possible for this value to not be set, and in that case would throw a notice.  Let's add a fallback of `unknown` if not set to future-proof.